diff --git a/dist/commonjs/index.d.ts b/dist/commonjs/index.d.ts
index 1556f330559c1ec17e0e7ad43bb49fe8337034c9..94f336ee1c682aa8d2fcb18e8cfa72e4b28b2cb6 100644
--- a/dist/commonjs/index.d.ts
+++ b/dist/commonjs/index.d.ts
@@ -86,6 +86,7 @@ export declare const minimatch: {
     escape: (s: string, { windowsPathsNoEscape, magicalBraces, }?: Pick<MinimatchOptions, "windowsPathsNoEscape" | "magicalBraces">) => string;
     unescape: (s: string, { windowsPathsNoEscape, magicalBraces, }?: Pick<MinimatchOptions, "windowsPathsNoEscape" | "magicalBraces">) => string;
 };
+export default minimatch;
 export type Sep = '\\' | '/';
 export declare const sep: Sep;
 export declare const GLOBSTAR: unique symbol;
diff --git a/dist/commonjs/index.js b/dist/commonjs/index.js
index 5a0ba2a87fb0365a959223919e82e6ac32dc49f8..566c7a9fbc0b3de5558f4db584e94adddde7af62 100644
--- a/dist/commonjs/index.js
+++ b/dist/commonjs/index.js
@@ -15,6 +15,7 @@ const minimatch = (p, pattern, options = {}) => {
     return new Minimatch(pattern, options).match(p);
 };
 exports.minimatch = minimatch;
+exports.default = minimatch;
 // Optimized checking for the most common glob patterns.
 const starDotExtRE = /^\*+([^+@!?\*\[\(]*)$/;
 const starDotExtTest = (ext) => (f) => !f.startsWith('.') && f.endsWith(ext);
diff --git a/dist/esm/index.d.ts b/dist/esm/index.d.ts
index 1556f330559c1ec17e0e7ad43bb49fe8337034c9..94f336ee1c682aa8d2fcb18e8cfa72e4b28b2cb6 100644
--- a/dist/esm/index.d.ts
+++ b/dist/esm/index.d.ts
@@ -86,6 +86,7 @@ export declare const minimatch: {
     escape: (s: string, { windowsPathsNoEscape, magicalBraces, }?: Pick<MinimatchOptions, "windowsPathsNoEscape" | "magicalBraces">) => string;
     unescape: (s: string, { windowsPathsNoEscape, magicalBraces, }?: Pick<MinimatchOptions, "windowsPathsNoEscape" | "magicalBraces">) => string;
 };
+export default minimatch;
 export type Sep = '\\' | '/';
 export declare const sep: Sep;
 export declare const GLOBSTAR: unique symbol;
diff --git a/dist/esm/index.js b/dist/esm/index.js
index 3175327db5efbf7e59e6294a767bec9835f1e4cf..bfbe2ab9775fdd2663d06720215b4f9b56fa9815 100644
--- a/dist/esm/index.js
+++ b/dist/esm/index.js
@@ -11,6 +11,7 @@ export const minimatch = (p, pattern, options = {}) => {
     }
     return new Minimatch(pattern, options).match(p);
 };
+export default minimatch;
 // Optimized checking for the most common glob patterns.
 const starDotExtRE = /^\*+([^+@!?\*\[\(]*)$/;
 const starDotExtTest = (ext) => (f) => !f.startsWith('.') && f.endsWith(ext);
